composer/src/main/kotlin/com/gojuno/composer/Apk.kt

```diff
@@ -61,4 +62,7 @@ fun parseTestRunner(testApkPath: String): TestRunner =
                 }
                 .toSingle()
                 .toBlocking()
-                .value()
\ No newline at end of file
+                .value()
+
+fun parseTests(testApkPath: String) : List<String> =
```

Comment:
Seems like we need to keep [TestMethod](https://github.com/linkedin/dex-test-parser/blob/master/parser/src/main/kotlin/com/linkedin/dex/parser/TestMethod.kt#L12) info, to filter out `@Ignore` before run and keep custom annotations info

Reply:
Ideally we'd have own TestMethod objects (we have some, I'm not sure they cover this use-case), so plugins (see below) could extend it with custom info and Dexparser lib would be just implementation detail rather than API everyone has to use